### PR TITLE
EA-2053 Set title according to language

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -280,35 +282,37 @@ export const setLinkElementData = ($el, manifest) => {
...
 
   if (duration === -1 && manifestJsonld.items[0].duration) {
     duration = manifestJsonld.items[0].duration;
   }
 };
 
-export const initialisePlayer = (mediaUrl, mediaMode) => {
-  let p = new EuropeanaMediaPlayer(playerWrapper, { manifest: mediaUrl }, { mode: 'player', manifest: mediaUrl });
+export const initialisePlayer = (mediaUrl, mediaMode, language) => {
+  let p = new EuropeanaMediaPlayer(playerWrapper, { manifest: mediaUrl }, { mode: 'player', manifest: mediaUrl, language });
```
Comment (pietervanleeuwen) [inline src/index.js:302]: Are you passing here only the language from the url parameter? The option should be set as `language: language` (see https://github.com/europeana/media-player#editor-and-language-example)

**Review by pietervanleeuwen — COMMENTED** [review]
Could you check my comment https://github.com/europeana/media-embed-service/pull/12/files#r423039913

File: src/index.js (hunk 1 above)
Comment (andyjmaclean) [reply src/index.js:302]: That fails linting, with the recommendation to just pass it as "language" (ES6 syntax)

Comment (pietervanleeuwen) [reply src/index.js:302]: Ok, then I have no further comments

**Pull request merged**